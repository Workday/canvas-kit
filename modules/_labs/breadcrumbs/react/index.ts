import Breadcrumbs from './lib/Breadcrumbs';

export default Breadcrumbs;
export {Breadcrumbs};
export * from './lib/Breadcrumbs';
