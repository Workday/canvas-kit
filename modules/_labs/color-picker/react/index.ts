export {default, default as ColorPicker, ColorPickerProps} from './lib/ColorPicker';
export {Swatch} from './lib/parts/Swatch';
