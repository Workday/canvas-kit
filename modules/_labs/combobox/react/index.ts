import Combobox from './lib/Combobox';

export default Combobox;
export {Combobox};
export * from './lib/Combobox';
