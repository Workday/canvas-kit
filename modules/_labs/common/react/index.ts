export * from './lib/Flex';
export * from './lib/Box';
