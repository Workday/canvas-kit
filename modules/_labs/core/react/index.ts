import type from './lib/type';
import space from './lib/space';
export default type;
export {type, space};
export * from './lib/type';
export * from './lib/StaticStates';
