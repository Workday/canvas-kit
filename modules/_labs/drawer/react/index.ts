import Drawer from './lib/Drawer';
import DrawerHeader from './lib/DrawerHeader';
export {Drawer, DrawerHeader};
export default Drawer;
export * from './lib/Drawer';
