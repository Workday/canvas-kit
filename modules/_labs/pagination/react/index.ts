import Pagination from './lib/Pagination';

export default Pagination;
export {Pagination};
export * from './lib/Pagination';
