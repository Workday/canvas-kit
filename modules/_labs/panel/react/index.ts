import Panel from './lib/Panel';

export default Panel;
export {Panel};
export * from './lib/Panel';
