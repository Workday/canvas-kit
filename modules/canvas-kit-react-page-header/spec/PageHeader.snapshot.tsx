import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PageHeader from '../lib/PageHeader';

describe('Page Header Snapshots', () => {
  test('renders a basic PageHeader as expected', () => {
    const component = renderer.create(<PageHeader title="Test Page Header" />);
    expect(component).toMatchSnapshot();
  });
  test('renders a marketing context PageHeader as expected', () => {
    const component = renderer.create(
      <PageHeader title="Test Marketing Page Header" marketing={true} />
    );
    expect(component).toMatchSnapshot();
  });
});
