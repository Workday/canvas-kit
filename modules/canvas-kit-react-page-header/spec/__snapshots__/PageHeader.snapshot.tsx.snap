// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Header Snapshots renders a basic PageHeader as expected 1`] = `
.emotion-6 {
  height: 84px;
  background-image: linear-gradient(to bottom right,#0875e1,#005cb9);
  color: #ffffff;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  height: 100%;
  overflow: hidden;
  padding: 0 16px;
}

@media (min-width:768px) {
  .emotion-4 {
    padding: 0 40px;
  }
}

.emotion-0 {
  font-size: 28px;
  line-height: 36px;
  font-weight: 500;
  color: #ffffff;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  padding: 12px 0;
  margin: 0;
  white-space: nowrap;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: 32px;
}

.emotion-2 > *:not(:last-child) {
  margin-right: 8px;
}

<header
  className="emotion-6 emotion-7"
>
  <div
    className="emotion-4 emotion-5"
    title="Test Page Header"
  >
    <h2
      className="emotion-0 emotion-1"
    >
      Test Page Header
    </h2>
    <div
      className="emotion-2 emotion-3"
    />
  </div>
</header>
`;

exports[`Page Header Snapshots renders a marketing context PageHeader as expected 1`] = `
.emotion-6 {
  height: 84px;
  background-image: linear-gradient(to bottom right,#0875e1,#005cb9);
  color: #ffffff;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

.emotion-0 {
  font-size: 28px;
  line-height: 36px;
  font-weight: 500;
  color: #ffffff;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  padding: 12px 0;
  margin: 0;
  white-space: nowrap;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: 32px;
}

.emotion-2 > *:not(:last-child) {
  margin-right: 8px;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  height: 100%;
  overflow: hidden;
  padding: 0 16px;
  box-sizing: border-box;
  margin: 0 auto;
  width: 100%;
  max-width: 1440px;
}

@media (min-width:768px) {
  .emotion-4 {
    padding: 0 40px;
  }
}

@media (min-width:992px) {
  .emotion-4 {
    padding: 0 64px;
  }
}

@media (min-width:1200px) {
  .emotion-4 {
    padding: 0 100px;
  }
}

<header
  className="emotion-6 emotion-7"
>
  <div
    className="emotion-4 emotion-5"
    title="Test Marketing Page Header"
  >
    <h2
      className="emotion-0 emotion-1"
    >
      Test Marketing Page Header
    </h2>
    <div
      className="emotion-2 emotion-3"
    />
  </div>
</header>
`;
